src/hyperfine/error.rs

```diff
@@ -26,9 +35,11 @@ impl Error for ParameterScanError {
     fn description(&self) -> &str {
         match *self {
             ParameterScanError::ParseIntError(ref e) => e.description(),
+            ParameterScanError::ParseDecimalError(ref e) => e.description(),
             ParameterScanError::EmptyRange => "Empty parameter range",
             ParameterScanError::TooLarge => "Parameter range is too large",
             ParameterScanError::ZeroStep => "Zero is not a valid parameter step",
+            ParameterScanError::StepRequired => "Step is required when range bounds are floats",
```

Comment:
Maybe: *"A step size is required when the range bounds are floating point numbers. The step size can be specified with the '--parameter-step-size' parameter"*

Reply:
This message is much better than the last. I have updated it.